### PR TITLE
removed duplicates from tutorials

### PR DESCRIPTION
**Pull request by polkhe**
removed copies

### CODE REVIEW

File: topics/admin/metadata.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ material:
     name: "dev-to-production"
     zenodo_link: ""
     galaxy_tour: ""
-    hands_on: "github"
```
Comment (bebatut) [inline topics/admin/metadata.yaml]: Can you let with github and add "enable: false"? This tutorial is not ready to be online

File: topics/chip-seq/metadata.yaml [hunk 2]
```diff
@@ -63,7 +63,7 @@ material:
     name: "chip-seq"
     zenodo_link: ""
     galaxy_tour: ""
-    hands_on: "github"
+    hands_on: "yes"
```
Comment (bebatut) [inline topics/chip-seq/metadata.yaml]: Can you let "github"? The tutorial is not fitting the templating requirements

File: topics/sequence-analysis/tutorials/de-novo-rad-seq/metadata.yaml [hunk 3]
```diff
@@ -1,180 +0,0 @@
...
-    email: "yvan.le-bras@mnhn.fr"
-
-contributors:
--
-    name: "Yvan Le Bras"
-    github_username: "yvanlebras"
-    email: "yvan.le-bras@mnhn.fr"
-
--
-    authors: "Amores, A., et al. 2011"
```
Comment (bebatut) [inline topics/sequence-analysis/tutorials/de-novo-rad-seq/metadata.yaml]: Can you move the references here in the topic `metadata.yaml`?

File: topics/sequence-analysis/tutorials/genetic-map-rad-seq/metadata.yaml [hunk 4]
```diff
@@ -1,181 +0,0 @@
...
-    github_username: "yvanlebras"
-    email: "yvan.le-bras@mnhn.fr"
-
-contributors:
--
-    name: "Yvan Le Bras"
-    github_username: "yvanlebras"
-    email: "yvan.le-bras@mnhn.fr"
-
-references:
```
Comment (bebatut) [inline topics/sequence-analysis/tutorials/genetic-map-rad-seq/metadata.yaml]: Can you move the references to the topic `metadata.yaml`?

File: topics/sequence-analysis/tutorials/quality-control/metadata.yaml [hunk 5]
```diff
@@ -1,72 +0,0 @@
...
-    github_username: "bebatut"
-    email: "berenice.batut@gmail.com"
-
-contributors:
--
-    name: "Bérénice Batut"
-    github_username: "bebatut"
-    email: "berenice.batut@gmail.com"
-
-references:
```
Comment (bebatut) [inline topics/sequence-analysis/tutorials/quality-control/metadata.yaml]: Can you move the reference to the topic `metadata.yaml`?

File: topics/sequence-analysis/tutorials/ref-based-rad-seq/metadata.yaml [hunk 6]
```diff
@@ -1,187 +0,0 @@
...
-    github_username: "yvanlebras"
-    email: "yvan.le-bras@mnhn.fr"
-
-contributors:
--
-    name: "Yvan Le Bras"
-    github_username: "yvanlebras"
-    email: "yvan.le-bras@mnhn.fr"
-
-references:
```
Comment (bebatut) [inline topics/sequence-analysis/tutorials/ref-based-rad-seq/metadata.yaml]: Can you move the reference to the topic `metadata.yaml`?

File: topics/sequence-analysis/tutorials/ref-based-rad-seq/metadata.yaml [hunk 7]
```diff
@@ -1,187 +0,0 @@
...
-    type: "internal"
-    link: "/NGS-mapping"
--
-    title: "Trackster"
-    type: "internal"
-    link: "/Introduction/Trackster"
-
-maintainers:
--
-    name: "Yvan Le Bras"
```
Comment (bebatut) [inline topics/sequence-analysis/tutorials/ref-based-rad-seq/metadata.yaml]: Can you add him as maintainers and contributors for the topic?

Comment (bebatut) [inline topics/sequence-analysis/tutorials/ref-based-rad-seq/metadata.yaml]: And check also for the other topic?

**Comment by bebatut** [comment]
Or better.
Can you move the contributors after `key_points` for each tutorial in the topic `metadata.yaml` file (and then remove the global  `contributors` section for the topic) and the maintainers for the topic?
I am not sure if it is clear. So an example:

```
name: "Name of the topic"
type: "use"
title: "topic_name"
summary: "Description of the topic"
requirements:
-
    title: "Galaxy introduction"
    type: "internal"
    link: "/Introduction/"
-
    title: "tutorial1"
    type: "tutorial"
    name: "tutorial1"
    zenodo_link: ""
    galaxy_tour: ""
    hands_on: "yes"
    slides: "no"
    questions:
    - ""
    - ""
    objectives:
    - ""
    - ""
    - ""
    time_estimation: "1d/3h/6h"
    key_points:
    - ""
    - ""
    - ""
    - "..."
    contributors:
    -
        name: "Contributor"
        github_username: "contributor"
        email: "contributor@galaxy.org"
    -
        name: "Contributor"
        github_username: "contributor"
        email: "contributor@galaxy.org"
-
    title: "tutorial2"
    type: "tutorial"
    name: "tutorial2"
    zenodo_link: ""
    galaxy_tour: ""
    hands_on: "yes"
    slides: "no"
    questions:
    - ""
    - ""
    objectives:
    - ""
    - ""
    - ""
    time_estimation: "1d/3h/6h"
    key_points:
    - ""
    - ""
    - ""
    - "..."
    contributors:
    -
        name: "Contributor"
        github_username: "contributor"
        email: "contributor@galaxy.org"
    -
        name: "Contributor"
        github_username: "contributor"
        email: "contributor@galaxy.org"

maintainers:
-
    name: "Maintainer"
    github_username: "maintainer"
    email: "maintainer@galaxy.org"
-
    name: "Maintainer"
    github_username: "maintainer"
    email: "maintainer@galaxy.org"
```

File: topics/sequence-analysis/metadata.yaml [hunk 8]
```diff
@@ -131,22 +164,291 @@ material:
...
+    title: "Multiplexed shotgun genotyping for rapid and efficient genetic mapping"
+    link: "https://www.ncbi.nlm.nih.gov/pubmed/21233398"
+-
+    authors: "Elshire, R. J., et al. 2011"
+    title: "A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species"
+    link: "http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0019379"
+-
+    authors: "Rigola, D., et al. 2009"
+    title: "High-Throughput Detection of Induced Mutations and Natural Variation Using KeyPoint� Technology"
+    links: (http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004761"
```
Comment (shiltemann) [inline topics/sequence-analysis/metadata.yaml]: think `(` should be `"` at start here?

Comment (bebatut) [inline topics/sequence-analysis/metadata.yaml]: Fixed

**Comment by bebatut** [comment]
@bgruening Ready to merge this one?

**Comment by willdurand** [comment]
maybe squashing these commits would make sense, WDYT? The commit messages are not that meaningful..

**Comment by bgruening** [comment]
Ok we all have know this. This one will be scary but we need to merge it sooner or later. If we dropped some contributors I will take the responsibility and all blame goes to me.

Thanks @polkhe!

**Pull request merged**